### ISSUE TITLE
How to interpret the profiler statistics here

### ISSUE THREAD

**Issue by PC09** [PyTorchLightning/pytorch-lightning#2258]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
I had an existing code in Pytorch which was taking ~100mins for an epoch. So I converted it to PyTorch-lightning module format to get the timings by the profiler. Surprisingly now it took ~60 mins to train but the statistics which I got are weird. The time (if it's in sec) is very less which doesn't sum up to 60 mins. Can anyone explain why is it so?

Epoch 1: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3000/3000 [1:02:09<00:00,  1.24s/it, loss=0.109, v_num=1]

Profiler Report

Action                  |  Mean duration (s)    |  Total time (s)
-----------------------------------------------------------------
on_train_start          |  0.0015837            |  0.0015837
on_epoch_start          |  0.00032781           |  0.00032781
get_train_batch         |  0.0094398            |  28.329
on_batch_start          |  1.8537e-05           |  0.055612
model_forward           |  0.03761              |  112.83
model_backward          |  0.025818             |  77.455
on_after_backward       |  3.9733e-06           |  0.01192
optimizer_step          |  0.024734             |  74.201
on_batch_end            |  0.0013818            |  4.1453
on_epoch_end            |  3.6526e-05           |  3.6526e-05
on_train_end            |  0.00052313           |  0.00052313

#### Code 

train_loader = get_loader(config.train_data_path, 'Train')
model = Network()
trainer = pl.Trainer(max_epochs=1, gpus=2, profiler=True, distributed_backend='dp')
trainer.fit(model, train_loader) 

#### What have you tried?
I tried to figure out possible bugs but just implemented the basic code as given in the examples section
#### What's your environment?

OS- linux, using conda
